docs/public/content/demo/demo_guide.md

```diff
@@ -0,0 +1,134 @@
++++
+title = "mFlow Demo"
+insert_anchor_links = "right"
+weight = 1
++++
+
+## Introduction
+
+Demo can be accessed at: [https://irims-demo.msupply.org:50000/](https://irims-demo.msupply.org:50000/)
+
+This demo allows you to use the **mFlow Application Manager** system to experience the following workflow:
+
+1. **Create a user account**
+
+   This is an automated process that requires email verification.
+
+2. **Register a company**
+
+   A reviewer will approve this application, which will create your organisation in the system.
+
+3. **Apply for a license for your company to register products**
```

Comment:
Should be `licence`? If yes, there are a couple of places to change

Reply:
Yeah, I've been quite inconsistent with this. Looks like "licence" is more standard in NZ (https://www.nzta.govt.nz/driver-licences/), so will go with that. Maybe change it on your template too?